larray/core/array.py

```diff
@@ -1002,8 +1002,12 @@ def to_frame(self, fold_last_axis_name=False, dropna=None):
                 tmp = axes_names[-1] if axes_names[-1] is not None else ''
                 if self.axes[-1].name:
                     axes_names[-1] = "{}\\{}".format(tmp, self.axes[-1].name)
-
-            index = pd.MultiIndex.from_product(self.axes.labels[:-1], names=axes_names)
+            # since version 0.21 of pandas, the MultiIndex constructors no longer squeezes
```

Comment:
Unsure we need the comment. If we change the code by inadvertence, we will see tests failing, right?